codecs/MWV.js

```diff
@@ -45,12 +45,16 @@ Field Number:
 
 var Codec = require('../lib/NMEA0183');
 
+Math.radians = function(degrees) {
+  return degrees * Math.PI / 180;
+};
```

Comment:
@fabdrol thanks.  If you have the time, go ahead with it.  This was a good exercise, and I'm starting to better know my way around the system.